packages/zosfiles/src/methods/create/Create.ts

```diff
@@ -70,6 +70,9 @@ export class Create {
             case CreateDataSetTypeEnum.DATA_SET_CLASSIC:
                 tempOptions = { ...CreateDefaults.DATA_SET.CLASSIC, ...tempOptions };
                 break;
+            case CreateDataSetTypeEnum.DATA_SET_BLANK:
+                tempOptions = { ...CreateDefaults.DATA_SET.BLANK, ...tempOptions };
```

Comment:
Should we add a unit test to `packages/zosfiles/__tests__/__unit__/methods/create/Create.unit.test.ts`?

Reply:
I added one...please lmk if it is enough. Since the blank template doesn't contain any properties, we don't really need to test overriding of template properties.